1. A sample of Alzheimer’s patients is tested to assess the amount of time in stage IV sleep. It has been hypothesized that individuals sufferering from Alzheimer's disease may spend less time per night in the deeper stages of sleep. Number of minutes spent is Stage IV sleep is recorded for sixty-one patients. The sample produced a mean of 48 minutes (S=14 minutes) of stage IV sleep over a 24 hour period of time. Compute a 95 percent confidence interval for this data. What does this information tell you about a particular individual's (an Alzheimer's patient) stage IV sleep?<br>
<b> Answer: </b><br>
Sample size n = 61<br>
Mean of number of minutes spent (x bar)= 48 Minutes<br>
Sample Standard deviation SD= 14 Minutes<br>
Confidence Interval (1-alpha) = 95%<br>
z value for alpha/2 is 1.96<br>

In [6]:
import math
-2/(3/math.sqrt(15))

-2.581988897471611

In [119]:
import scipy.stats as stats
stats.norm.interval(0.95,
                 loc = 48,
                 scale= 14/math.sqrt(61)) 

(44.48673257294102, 51.51326742705898)

Hence, 95% Confidence Interval is between 44.49 Minutes and 51.51 Minutes.
i.e. This tell that at stage IV an alzheimer's patient deep sleep is of 44.49 minutes to 51.51 minutes.
Still there is 5% chance that above inference may be wrong.

2. A university wants to know more about the knowledge of students regarding international events. They are concerned that their students are uninformed in regards to new from other countries. A standardized test is used to assess students knowledge of world events (national reported mean=65, S=5). A sample of 30 students are tested (sample mean=58, Standard Error=3.2). Compute a 99 percent confidence interval based on this sample's data. How do these students compare to the national sample?<br>
<b>Answer:</b><br>
Sample size N=30<br>
Sample Mean x bar=58<br>
Standard Error=3.2<br>
Confidence Interval=99<br>
1-alpha=0.99

In [108]:
import scipy.stats as stats
stats.t.interval(0.99,
                 df = 29,
                 loc = 58,
                 scale= 3.2) 

(49.17956510825493, 66.82043489174507)

Hence as with 99% Confidence Interval the Students knowledge test score is between 49.18 and 66.82, which includes the National reported mean of 65. Hence even though the score of college is lower, it is still in line with other colleges across the world.

3. A sample of students from an introductory psychology class were polled regarding the number of hours they spent studying for the last exam. All students anonymously submitted the number of hours on a 3 by 5 card. There were 24 individuals in the one section of the course polled. The data was used to make inferences regarding the other students taking the course. There data are below:<br>
Compute a 95 percent confidence interval. What does this tell us?<br>
<table>
    <tr>
        <td>4.5</td>
        <td>22</td>
        <td>7</td>
        <td>14.5</td>
        <td>9</td>
        <td>9</td>
        <td>3.5</td>
        <td>8</td>
        <td>11</td>
        <td>7.5</td>
        <td>18</td>
        <td>20</td>
    </tr>
    <tr>
        <td>7.5</td>
        <td>9</td>
        <td>10.5</td>
        <td>15</td>
        <td>19</td>
        <td>2.5</td>
        <td>5</td>
        <td>9</td>
        <td>8.5</td>
        <td>14</td>
        <td>20</td>
        <td>8</td>
    </tr>
</table>

In [129]:
import numpy as np
import scipy.stats as stats

studentsData=np.array([4.5,22,7,14.5,9,9,3.5,8,11,7.5,18,20,7.5,9,10.5,15,19,2.5,5,9,8.5,14,20,8])
print('Mean Time:',studentsData.mean(),'\nStandard Daviation:',studentsData.std(),'\nStandard Error',scipy.stats.sem(studentsData))
stats.t.interval(0.95,
                 df = 24,
                 loc = studentsData.mean(),
                 scale= scipy.stats.sem(studentsData)) 

Mean Time: 10.916666666666666 
Standard Daviation: 5.480394349152461 
Standard Error 1.142741216515229


(8.55816471358783, 13.275168619745502)

Hence the confidence interval for the data is between 8.56 hours and 13.27 hours.
Means we are 95% sure that on an average students spents 8.56 hours to 13.27 hours for the last exams.